### PR TITLE
Added centos7.3 x86_64 openstack

### PR DESCRIPTION
**Pull request by knightsamar**

### CODE REVIEW

File: centos-7.3-x86_64-openstack.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+  ],
+  "provisioners": [
+    {
+      "environment_vars": [
+        "CHEF_VERSION={{user `chef_version`}}"
+      ],
+      "execute_command": "echo 'centos' | {{.Vars}} sudo -S -E bash '{{.Path}}'",
+      "scripts": [
+        "scripts/centos/osuosl.sh",
```
Comment (hodgesto) [inline centos-7.3-x86_64-openstack.json]: Wondering what exactly is contained in here

**Comment by DesertBeagle** [comment]
@hodgesto `scripts/common/chef.sh` is already in the scripts section.

**Comment by knightsamar** [comment]
So @DesertBeagle please add those scripts to this PR.

**Comment by DesertBeagle** [comment]
@knightsamar Added. I think we should alter either the disk size, or change the flavor that taster uses.

File: centos-7.3-x86_64-openstack.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "builders": [
+    {
+      "boot_command": [
+        "<tab> text ks=http://{{ .HTTPIP }}:{{ .HTTPPort }}/centos-7.3/ks-openstack.cfg<enter><wait>"
```
Comment (knightsamar) [inline centos-7.3-x86_64-openstack.json]: Can you please add this file too to the PR ?

File: centos-7.3-x86_64-openstack.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      ],
+      "accelerator": "kvm",
+      "boot_wait": "10s",
+      "disk_size": 2048,
```
Comment (knightsamar) [inline centos-7.3-x86_64-openstack.json]: The disk size is smaller than the one in https://github.com/osuosl/packer-templates/blob/master/centos-7.3-ppc64le-openstack.json#L13 so I don't think that's the root cause of this inflated image.

File: centos-7.3-x86_64-openstack.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      "execute_command": "echo 'centos' | {{.Vars}} sudo -S -E bash '{{.Path}}'",
+      "scripts": [
+        "scripts/centos/osuosl.sh",
+        "scripts/centos/fix-slow-dns.sh",
+        "scripts/common/sshd.sh",
+        "scripts/common/vmtools.sh",
+        "scripts/common/chef.sh",
+        "scripts/centos/openstack.sh",
+        "scripts/centos/cleanup.sh",
+        "scripts/common/minimize.sh"
```
Comment (knightsamar) [inline centos-7.3-x86_64-openstack.json]: Are all these already in the repo ? If not please add to this PR.

File: centos-7.3-x86_64-openstack.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      ],
+      "type": "shell"
+    }
+  ],
+  "variables": {
+    "chef_version": "provisionerless",
+    "mirror": "http://centos.osuosl.org"
+  }
```
Comment (knightsamar) [inline centos-7.3-x86_64-openstack.json]: Please add an `image_name` variable like https://github.com/osuosl/packer-templates/blob/master/centos-7.3-ppc64le-openstack.json#L55

**Comment by DesertBeagle** [comment]
@knightsamar Added the variable name! Those other scripts are already present!

**Comment by DesertBeagle** [comment]
[CentOS x86_64 image](https://trello.com/c/IZU0Wdd2/466-centos-x8664-image)

File: http/centos-7.3/ks-openstack.cfg [hunk 6]
```diff
@@ -0,0 +1,86 @@
+install
```
Comment (knightsamar) [inline http/centos-7.3/ks-openstack.cfg]: So following the convention that we have started using in this template, can you please rename this file to `http/centos-7.3/ks-x86_64.cfg` ?

Comment (DesertBeagle) [inline http/centos-7.3/ks-openstack.cfg]: Okay!

**Comment by DesertBeagle** [comment]
@knightsamar Renamed!

**Comment by knightsamar** [comment]
Does it build ?

**Comment by DesertBeagle** [comment]
@knightsamar Now it does!

**Comment by knightsamar** [comment]
I cannot approve this technically, but i think it's good to squash into 1 commit and merge.
Unless @hodgesto has anything to add.

**Comment by hodgesto** [comment]
It looks good, I can't test it fully because I don't have WiFi on my laptop here but after quite a bit of looking over I say it's good.

**Comment by knightsamar** [comment]
Merging this fixed #18

**Pull request merged**